lib/juttle-engine.js

```diff
@@ -40,6 +40,8 @@ function run(opts, ready) {
 
     service.addRoutes(app, service_opts);
 
+    opts.host = opts.host || '0.0.0.0';
+
```

Comment:
This isn't a sensible default for the juttle-viewer configuration since `juttleServiceHost` can't point to 0.0.0.0.

It does however make sense as a default when creating the express server.